tool/webqa-process-schemaorg.js

```diff
@@ -1053,6 +998,18 @@ module.exports = {
             help: 'Enable manual annotations.',
             defaultValue: false
         });
+        parser.addArgument('--always-base-canonical', {
+            nargs: 0,
+            action: 'storeTrue',
+            help: `Always generate base canonical`,
+            defaultValue: true
+        });
+        parser.addArgument('--no-bad-base-canonical', {
+            nargs: 0,
+            action: 'storeFalse',
+            help: `Do not always generate base canonical`,
+            dest: `always_base_canonical`,
+        });
```

Comment:
By convention, if you have a pair of boolean flags, the second one is `--no-` the first one, so it's obvious they are paired.